### PR TITLE
[Transform] available caps without peer

### PR DESCRIPTION
**Pull request by jaeyun-jung**
tensor-transform cannot determine the caps, this element can set out-caps by using incoming caps. 
Remove unnecessary query for peer caps and update types of out-caps if input has tensors.

### CODE REVIEW

**Comment by jaeyun-jung** [comment]
@niley7464 please check this PR after clear your current issue :)

**Review by anyj0527 — APPROVED** [review]
LGTM, cool!

**Review by niley7464 — APPROVED** [review]
LGTM!!
Thank you :)

File: gst/nnstreamer/elements/gsttensor_transform.c [hunk 1]
```diff
@@ -2041,13 +2041,10 @@ gst_tensor_transform_transform_caps (GstBaseTransform * trans,
...
-    }
 
-    /* remove `types` field from caps */
-    if (is_types_not_fixed) {
-      GstStructure *s;
-      for (j = 0; j < gst_caps_get_size (result_aux); ++j) {
-        s = gst_caps_get_structure (result_aux, j);
+      /* remove `types` field from caps */
+      if (is_types_not_fixed) {
+        GstStructure *s = gst_caps_get_structure (result_aux, 0);
```
Comment (myungjoo) [inline gst/nnstreamer/elements/gsttensor_transform.c:2047]: How do you sure that result_aux will have only one GstStructure?

Comment (jaeyun-jung) [reply gst/nnstreamer/elements/gsttensor_transform.c:2047]: The caps 'result_aux' is created in the loop (line 2010), and new capability is appended once (2040~2043).

**Review by myungjoo — COMMENTED** [review]
If you can add a test case (a gst pipeline desc) that runs with this commit (a pipeline that can be negotiated only after this commit), it'd be fantastic. It will also provide a regression preventor.

**Review by myungjoo — APPROVED** [review]

**Pull request merged**